### DISCUSSION TITLE
关于二次分发PCL2

### DISCUSSION THREAD

**Discussion by Chun-QiuCC** [Meloong-Git/PCL#1263] [提问] [answered]
UP您好，我希望制作一个启动器软件的镜像站，请问我可以在我的网站上提供您制作的PCL启动器正式版的下载吗？ （镜像站位于中国大陆境外）

**Comment by LTCatt** [comment, marked as answer]
可以

**Comment by Chiloven945** [comment]
想看看镜像站（）

**Reply by naiximcn** [reply]
他的视频
https://www.bilibili.com/video/av991996165/
他的镜像站
https://mcl.mclive.org/

无意间刷到的 感觉很无聊 毫无意义

**Comment by 212112111zwl** [comment]
做完了给我们看看~

**Reply by naiximcn** [reply]
他的视频
https://www.bilibili.com/video/av991996165/
他的镜像站
https://mcl.mclive.org/

无意间刷到的 感觉很无聊 毫无意义

**Reply by Chun-QiuCC** [reply]
> 他的视频 https://www.bilibili.com/video/av991996165/ 他的镜像站 https://mcl.mclive.org/
> 
> 无意间刷到的 感觉很无聊 毫无意义

请问为什么你认为无聊的东西就是毫无意义？

**Comment by Chiloven945** [comment]
不理解也没必要这样说，做的事情总是有原因的
________________________________
发件人: NaixiNana ***@***.***>
发送时间: 2023年3月10日 21:57
收件人: Hex-Dragon/PCL2 ***@***.***>
抄送: Chiloven945 ***@***.***>; Comment ***@***.***>
主题: Re: [Hex-Dragon/PCL2] 关于二次分发PCL2 (Discussion #1263)


他的视频
https://www.bilibili.com/video/av991996165/
他的镜像站
https://mcl.mclive.org/

无意间刷到的 感觉很无聊 毫无意义

―

**Comment by Chiloven945** [comment]
我倒是有个建议，你可以加个详情页，介绍启动器什么的（
